### ISSUE TITLE
Wind turbine crash

### ISSUE THREAD

**Issue by weizhuowang** [mekanism/Mekanism#6057]
Hi,

When I placed the wind turbine, my client crashed. However, the turbine was still successfully placed and my server did not crash. After that, whenever I come too close to the wind turbine, the game will crash with the following logs.

```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 20-4-13 上午12:51
Description: Unexpected error

java.lang.IllegalAccessError: tried to access class net.minecraft.client.renderer.model.ModelRenderer$TexturedQuad from class mekanism.client.model.ExtendedModelRenderer
	at mekanism.client.model.ExtendedModelRenderer.renderWireFrame(ExtendedModelRenderer.java:38) ~[?:?] {re:classloading}
	at mekanism.client.model.ExtendedModelRenderer.render(ExtendedModelRenderer.java:22) ~[?:?] {re:classloading}
	at mekanism.generators.client.model.ModelWindGenerator.render(ModelWindGenerator.java:218) ~[?:?] {re:classloading}
	at mekanism.generators.client.model.ModelWindGenerator.renderWireFrame(ModelWindGenerator.java:214) ~[?:?] {re:classloading}
	at mekanism.generators.client.render.RenderWindGenerator.renderWireFrame(RenderWindGenerator.java:48) ~[?:?] {re:classloading}
	at mekanism.client.render.RenderTickHandler.configurableMachine(RenderTickHandler.java:248) ~[?:?] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_57_RenderTickHandler_configurableMachine_HighlightBlock.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:80) ~[eventbus-2.0.0-milestone.1-service.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:258) ~[eventbus-2.0.0-milestone.1-service.jar:?] {}
	at net.minecraftforge.client.ForgeHooksClient.onDrawBlockHighlight(ForgeHooksClient.java:168) ~[?:?] {re:classloading}
	at net.minecraft.client.renderer.WorldRenderer.func_228426_a_(WorldRenderer.java:1038) ~[?:?] {re:classloading,xf:fml:journeymap:WorldRenderer.markForRerender,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.func_228378_a_(GameRenderer.java:600) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:422) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:925) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:554) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(SourceFile:204) [?:?] {re:classloading}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51] {}
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:56) [forge-1.15.2-31.1.44.jar:31.1] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$407/653515020.call(Unknown Source) [forge-1.15.2-31.1.44.jar:31.1] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-5.0.0-milestone.4.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-5.0.0-milestone.4.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-5.0.0-milestone.4.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81) [modlauncher-5.0.0-milestone.4.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65) [modlauncher-5.0.0-milestone.4.jar:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at mekanism.client.model.ExtendedModelRenderer.renderWireFrame(ExtendedModelRenderer.java:38)
	at mekanism.client.model.ExtendedModelRenderer.render(ExtendedModelRenderer.java:22)
	at mekanism.generators.client.model.ModelWindGenerator.render(ModelWindGenerator.java:218)
	at mekanism.generators.client.model.ModelWindGenerator.renderWireFrame(ModelWindGenerator.java:214)
	at mekanism.generators.client.render.RenderWindGenerator.renderWireFrame(RenderWindGenerator.java:48)
	at mekanism.client.render.RenderTickHandler.configurableMachine(RenderTickHandler.java:248)
	at net.minecraftforge.eventbus.ASMEventHandler_57_RenderTickHandler_configurableMachine_HighlightBlock.invoke(.dynamic)
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:80)
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:258)
	at net.minecraftforge.client.ForgeHooksClient.onDrawBlockHighlight(ForgeHooksClient.java:168)
	at net.minecraft.client.renderer.WorldRenderer.func_228426_a_(WorldRenderer.java:1038)
	at net.minecraft.client.renderer.GameRenderer.func_228378_a_(GameRenderer.java:600)

-- Affected level --
Details:
	All players: 1 total; [ClientPlayerEntity['ASKKER'/1033, l='MpServer', x=-58.81, y=88.00, z=13.80]]
	Chunk stats: Client Chunk Cache: 3249, 2704
	Level dimension: DimensionType{minecraft:overworld}
	Level name: MpServer
	Level seed: 3708484580839949840
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: {}
	Level spawn location: World: (-45,71,-15), Chunk: (at 3,4,1 in -3,-1; contains blocks -48,0,-16 to -33,255,-1), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Level time: 353887 game time, 392235 day time
	Known server brands: 
	Level was modded: false
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Server brand: forge
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.world.ClientWorld.func_72914_a(ClientWorld.java:442)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:1764)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:573)
	at net.minecraft.client.main.Main.main(SourceFile:204)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:56)
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$407/653515020.call(Unknown Source)
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54)
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72)
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:81)
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65)

-- System Details --
Details:
	Minecraft Version: 1.15.2
	Minecraft Version ID: 1.15.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 2472044040 bytes (2357 MB) / 4294967296 bytes (4096 MB) up to 4294967296 bytes (4096 MB)
	CPUs: 4
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx4G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	ModLauncher: 5.0.0-milestone.4+67+b1a340b
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/eventbus-2.0.0-milestone.1-service.jar eventbus PLUGINSERVICE 
		/forge-1.15.2-31.1.44.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.15.2-31.1.44.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-2.0.4-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.15.2-31.1.44.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.15.2-31.1.44.jar runtimedistcleaner PLUGINSERVICE 
		/forge-1.15.2-31.1.44.jar fml TRANSFORMATIONSERVICE 
		/MixinBootstrap-1.0.2.jar mixinbootstrap TRANSFORMATIONSERVICE 
		/MixinBootstrap-1.0.2.jar mixin PLUGINSERVICE 
		/MixinBootstrap-1.0.2.jar mixinbootstrap PLUGINSERVICE 
	FML: 31.1
	Forge: net.minecraftforge:31.1.44
	FML Language Providers: 
		javafml@31.1
		minecraft@1
	Mod List: 
		forge-1.15.2-31.1.44-client.jar Minecraft {minecraft@1.15.2 DONE}
		Mekanism-1.15.2-9.9.19.411.jar Mekanism {mekanism@9.9.19 DONE}
		forge-1.15.2-31.1.44-universal.jar Forge {forge@31.1.44 DONE}
		MekanismTools-1.15.2-9.9.19.411.jar Mekanism: Tools {mekanismtools@9.9.19 DONE}
		MekanismAdditions-1.15.2-9.9.19.411.jar Mekanism: Additions {mekanismadditions@9.9.19 DONE}
		cfm-7.0.0-pre16-mc1.15.1.jar MrCrayfish's Furniture Mod {cfm@7.0.0-pre16 DONE}
		JourneyMap-Mod-1.15.2-Unlimited.jar Journeymap {journeymap@1.15.2-5.7.0beta1 DONE}
		MekanismGenerators-1.15.2-9.9.19.411.jar Mekanism: Generators {mekanismgenerators@9.9.19 DONE}
	Launched Version: 1.15.2-forge-31.1.44
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: GeForce GTX 1080/PCIe/SSE2 GL version 4.6.0 NVIDIA 432.00, NVIDIA Corporation
	GL Caps: Using framebuffer using OpenGL 3.0
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'
	Type: Client (map_client.txt)
	Resource Packs: vanilla
	Current Language: English (US)
	CPU: 4x Intel(R) Core(TM) i5-6600K CPU @ 3.50GHz
```

**Comment by ppoozl** [comment]
Same issue
Server Side LOG:
[13Apr2020 12:43:13.850] [Server thread/FATAL] [net.minecraft.world.World/]: ---- Minecraft Crash Report ----
// Would you like a cupcake?

Time: 4/13/20 12:43 PM
Description: Ticking block entity

java.lang.ArithmeticException: Division by zero
	at mekanism.api.math.FloatingLong.divideEquals(FloatingLong.java:329) ~[?:?] {re:classloading}
	at mekanism.api.math.FloatingLong.divide(FloatingLong.java:471) ~[?:?] {re:classloading}
	at mekanism.api.math.FloatingLong.divide(FloatingLong.java:503) ~[?:?] {re:classloading}
	at mekanism.common.tile.TileEntityThermalEvaporationController.lambda$createNewCachedRecipe$5(TileEntityThermalEvaporationController.java:244) ~[?:?] {re:classloading}
	at mekanism.api.recipes.cache.CachedRecipe.process(CachedRecipe.java:114) ~[?:?] {re:classloading}
	at mekanism.common.tile.TileEntityThermalEvaporationController.onUpdateServer(TileEntityThermalEvaporationController.java:167) ~[?:?] {re:classloading}
	at mekanism.common.tile.base.TileEntityMekanism.func_73660_a(TileEntityMekanism.java:519) ~[?:?] {re:classloading}
	at net.minecraft.world.World.func_217391_K(World.java:473) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:368) [?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:849) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:330) [?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:784) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:637) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_241] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at mekanism.api.math.FloatingLong.divideEquals(FloatingLong.java:329)
	at mekanism.api.math.FloatingLong.divide(FloatingLong.java:471)
	at mekanism.api.math.FloatingLong.divide(FloatingLong.java:503)
	at mekanism.common.tile.TileEntityThermalEvaporationController.lambda$createNewCachedRecipe$5(TileEntityThermalEvaporationController.java:244)
	at mekanism.api.recipes.cache.CachedRecipe.process(CachedRecipe.java:114)
	at mekanism.common.tile.TileEntityThermalEvaporationController.onUpdateServer(TileEntityThermalEvaporationController.java:167)
	at mekanism.common.tile.base.TileEntityMekanism.func_73660_a(TileEntityMekanism.java:519)

-- Block entity being ticked --
Details:
	Name: mekanism:thermal_evaporation_controller // mekanism.common.tile.TileEntityThermalEvaporationController
	Block: Block{mekanism:thermal_evaporation_controller}[active=true,facing=north]
	Block location: World: (-2392,24,-1447), Chunk: (at 8,1,9 in -150,-91; contains blocks -2400,0,-1456 to -2385,255,-1441), Region: (-5,-3; contains chunks -160,-96 to -129,-65, blocks -2560,0,-1536 to -2049,255,-1025)
	Block: Block{mekanism:thermal_evaporation_controller}[active=true,facing=north]
	Block location: World: (-2392,24,-1447), Chunk: (at 8,1,9 in -150,-91; contains blocks -2400,0,-1456 to -2385,255,-1441), Region: (-5,-3; contains chunks -160,-96 to -129,-65, blocks -2560,0,-1536 to -2049,255,-1025)
Stacktrace:
	at net.minecraft.world.World.func_217391_K(World.java:473)
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:368)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:849)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:330)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:784)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:637)
	at java.lang.Thread.run(Unknown Source)

**Comment by pupnewfster** [comment]
Fixed in 9.9.20
